### PR TITLE
QT: add menu option to open user data

### PR DESCRIPTION
**Pull request by BreadFish64**
small QOL improvement suggested by @chris062689 
image

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3457)

### CODE REVIEW

**Comment by BreadFish64** [comment]
I'm not sure exactly, but it gives the impression that the button will lead elsewhere, in the case of most of the menu options under the "file" menu, that's a file dialog or explorer window

**Comment by mailwl** [comment]
dislike

**Comment by wwylele** [comment]
@mailwl could you elaborate your reasoning? I simple "dislike" can hardly help us decide the design change.

**Comment by einstein95** [comment]
Like

**Comment by adityaruplaha** [comment]
@Subv It's called an "ellipsis" & indicates that there is another step after clicking this option for the thing described here to actually happen. Eg: 
When we save a file for the 1st time (save as...) we get the prompt for the filename, then the file gets save. Hence, there's an ellipsis. Idk what its doing here. Its a direct action.

I hope I explained well.

File: src/citra_qt/main.ui [hunk 1]
```diff
@@ -139,6 +140,11 @@
     <string>Load Symbol Map...</string>
    </property>
   </action>
+  <action name="action_Open_User_Data_Location">
+   <property name="text">
+    <string>Open User Data Location...</string>
```
Comment (adityaruplaha) [inline src/citra_qt/main.ui]: Remove the ellipsis ("..."). Its a direct action.

**Comment by adityaruplaha** [comment]
Please fix clang format as well.

**Comment by BreadFish64** [comment]
@adityaruplaha I'm well aware that clang-format is failing, master has been failing for about a month, don't worry about it.

**Comment by jroweboy** [comment]
I voiced my complaints about this on Discord so I might as well make them on here. I don't see a point to this PR. Please provide a clear reason why we should add this (ie what is the expected use case for the menu item) otherwise its just clutter.

If you want to provide access to the location that citra stores everything, then this isn't enough on linux https://github.com/citra-emu/citra/blob/master/src/common/file_util.cpp#L710 where we have different paths for the user folder and the config folder. If this is meant to let people manually edit the config, then this is also a bad idea, as it doesn't take you to the config location on linux and mac. If this is just meant to get you to the log location, I already have a button for "open log location" in the logging rewrite pr.

My suggestion was to add a button to the config somewhere that takes you to the config location **if thats something that we expect people to commonly need** but with controller config through the UI, I don't see this as necessary.

**Comment by BreadFish64** [comment]
I see your point :)

**Pull request closed without merging**